src/assets/HTMLAsset.js

```diff
@@ -12,6 +12,7 @@ const ATTRS = {
   src: [
     'script',
     'img',
+    'image',
```

Comment:
Should probably be part of the array of key: `'xlink:href'`

Reply:
I got that but this would only fix it for `<image src='' />` nor for <image xlink:href="myfile.svg" />

Reply:
Could you also add a test for this. To make sure it's fixed and doesn't cause anything else to fail